### ISSUE TITLE
model definition location

### ISSUE THREAD

**Issue by topepo** [business-science/gamsnip#8]
If you think that other packages might have engine definitions for `gam_mod()`, it would make sense to have the definition for that model live in `parsnip` and have the engines registered here.

**Comment by mdancho84** [comment]
I agree - what do you think: if we first get the package up and running as 0.1.0, and then in version 0.2.0 we transition the definition to parsnip and retain the engines? Just thinking about a good transition plan that doesn't slow down the development.

**Comment by topepo** [comment]
I would do it before it goes to CRAN. We'd have to soft-deprecate the old version and it would just be easier to move it before it is "official"

**Comment by mdancho84** [comment]
Hey @topepo We are back on `gammodels` (yes, we had one more name change). We'd like to begin the process of preparing the package for integration with `parsnip`. We can finalize the package on our end, but it sounds like you'd like the main `gen_additive_mod()` function to rest in `parnsip`. What's the best way to get started there?
